src/runtime/contrib/arm_compute_lib/acl_utils.cc

```diff
@@ -130,6 +130,8 @@ arm_compute::DataType MakeACLDataType(const DLDataType& data_type) {
     return arm_compute::DataType::F32;
   } else if (data_type.code == DLDataTypeCode::kDLUInt && data_type.bits == 8) {
     return arm_compute::DataType::QASYMM8;
+  } else if (data_type.code == DLDataTypeCode::kDLInt && data_type.bits == 8) {
+    return arm_compute::DataType::QASYMM8_SIGNED;
```

Comment:
Just curious why only asymmetric dtype is supported?

Reply:
I got it from the data types list here: https://github.com/ARM-software/ComputeLibrary/blob/aabef6c0584f06f4c0f4b61fb787d80374240619/src/core/CL/CLHelpers.cpp#L39

Reply:
After discussing this with @lhutton1, it looks like some of the buffers need to be symmetric as described on the https://www.tensorflow.org/lite/performance/quantization_spec. For Conv2D, Depthwise, Fully Connected layers, this could come handy. I wonder if the ACL errors out if the incorrect type is passed to the end APIs.